portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessdeliveryapi/CommentAPI.macro

```diff
@@ -87,12 +142,34 @@ definition {
 	macro getFieldValueOfBlogPostingComments {
 		Variables.assertDefined(parameterList = "${fieldName}");
 
-		if (!(isSet(responseToParse))) {
+		if (isSet(blogPostingId) && !(isSet(commentId))) {
```

Comment:
Maybe we can leave responseToParse so that we can call it in .testcase file like:
![image](https://user-images.githubusercontent.com/1308942/186813891-86162d4c-bb4d-4b83-a35a-aa85a3bde786.png)